ui/src/app/applications/components/application-create-panel/application-create-panel.tsx

```diff
@@ -231,13 +233,24 @@ export const ApplicationCreatePanel = (props: {
                                                 <p>SOURCE</p>
                                                 <div className='row argo-form-row'>
                                                     <div className='columns small-10'>
-                                                        <FormField
-                                                            formApi={api}
-                                                            label='Repository URL'
-                                                            qeId='application-create-field-repository-url'
-                                                            field='spec.source.repoURL'
-                                                            component={AutocompleteField}
-                                                            componentProps={{items: repos}}
+                                                        {false && (
```

Comment:
This `FormField` is not rendered due to `{false && (`, correct? Remove it please

Reply:
I think this was to silence a compiler error while I was debugging, forgot to remove it. Thank you for catching it!